html-forms/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Html Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div>
+        <label for="email">Email <br /></label>
+        <input type="text" name="email" id="email" />
+      </div>
+      <div>
+        <label for="why is coding cool?">Why is coding cool?<br /></label>
+        <select name="why is coding cool?" id="why is coding cool?">
+          <option value="select an option" selected>Select an Option</option>
+          <option value="because it's interesting">
+            Because it's interesting
+          </option>
+          <option value="because it's fun">Because it's fun</option>
+          <option value="because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label for="favorite language"
+          >Favorite Language (choose one) <br
+        /></label>
+        <input type="radio" id="html" name="language" value="html" />
+        <label for="html">HTML</label>
+        <input type="radio" id="css" name="language" value="css" />
+        <label for="css">CSS</label>
+        <input
+          type="radio"
+          id="javascript"
+          name="language"
+          value="javascript" />
+        <label for="javascript">Javascript</label>
+      </div>
+      <div>
+        <label for="would you like to sign up for more information?"
+          >Would you like to sign to for more information? <br
+        /></label>
+        <input type="checkbox" id="sign me up" />
+        <label for="sign me up">Sign me up!</label>
+      </div>
+      <div>
+        <label for="comments">Comments <br /></label>
+        <textarea name="comments" id="comments" cols="30" rows="10"></textarea>
+      </div>
+      <div>
+        <input type="submit" id="submit" />
+        <label for="submit"></label>
```

Comment:
instead of a label, I would recommend the `value` attribute for the input with the value of `Submit`